RootOutputer.cc

```diff
@@ -20,6 +21,7 @@ RootOutputer::RootOutputer(std::string const& iFileName, unsigned int iNLanes, C
   basketSize_{iConfig.basketSize_},
   splitLevel_{iConfig.splitLevel_}
 {
+  new TFileCacheWrite(&file_, iConfig.cacheSize_);
```

Comment:
I'd prefer this not be on all the time so we can test with and without it. How about
```suggestion
  if(iConfig.cacheSize_ > 0 ) { 
     new TFileCacheWrite(&file_, iConfig.cacheSize_);
  }
```